src/main/webapp/app/scores/exercise-scores.component.ts

```diff
@@ -136,6 +139,9 @@ export class ExerciseScoresComponent implements OnInit, OnDestroy {
                 return result.successful;
             case FilterProp.UNSUCCESSFUL:
                 return !result.successful;
+            case FilterProp.BUILD_FAILED:
+                // TODO: A boolean flag {buildFailed} on the result coming from the backend would be better
```

Comment:
You actually can't rely on this string for your check. This is a Bamboo specific string and might be different for other CI systems (With the Jenkins integration, we would get the first problem). I'll open an issue related to this since I saw that we are also using this check in other places.

Reply:
Yeah, I didn't come up with that check by myself, I just inspected how this distinction is done in the `result` component. It definitely feels very hacky, hence why I left the todo. I didn't want to address it in this PR since refactoring that would probably not just be a small side-fix but completely overshadow the actual goal of this PR.

Reply:
Will be solved with #1153

Reply:
As long as it is consistent between Bamboo and Jenkins, this is ok.
I guess we should keep 'No tests found' for compatibility reasons...